t/2_mwsrange.t

```diff
@@ -0,0 +1,15 @@
+use LaTeXML::Util::Test;
+use File::Copy qw(copy);
+
+# Ok i'll guess this is a realy dirty workaround, but hmmm
```

Comment:
Do you really want to check in this workaround? Looks quite messy...

Reply:
I found no less messy way to make the tests possible. Since latexml_tests is expecting the latemlc in the blib/scripts folder